jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -32,7 +37,8 @@ class Shutdown(object):
 
     def __init__(self):
         rospy.loginfo('Start shutdown node.')
-
+        self.client = actionlib.SimpleActionClient(
+            '/robotsound_jp', SoundRequestAction)
```

Comment:
Could you add a document for `/robotsound_jp'?

Reply:
Thanks! I updated the document.